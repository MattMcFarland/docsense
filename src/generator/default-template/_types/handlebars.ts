export interface BlockHelper {
  fn: (ctx: any) => any;
}
