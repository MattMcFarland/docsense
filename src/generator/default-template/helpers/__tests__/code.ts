import * as Handlebars from 'handlebars';

describe('hbs helpers > code', () => {
  test('matches snapshot', () => {
    const code = require('../code');
    Handlebars.registerHelper('code', code);
    const hbs = `<div>{{#code}}var foo = 'bar'{{/code}}</div>`;
    const template = Handlebars.compile(hbs);
    const result = template({});
    expect(result).toMatchSnapshot();
  });
});
