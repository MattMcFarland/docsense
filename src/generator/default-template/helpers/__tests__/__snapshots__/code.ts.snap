// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hbs helpers > code matches snapshot 1`] = `"<div><pre class=\\"ba b--black-20 lh-solid\\"><code class=\\"javascript\\"><span class=\\"hljs-selector-tag\\">var</span> foo = <span class=\\"hljs-string\\">'bar'</span></code></pre></div>"`;
